index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Contact us page</title>
+</head>
+<body>
+    <section>
+        
+    <div class="container">
+        <div class="contactinfo">
+          <div>
+           <h2>Contact info</h2> 
+            <ul class="info">
+                <li>
+                    <span>
+                        <img src="img1/kindpng_1084606.png" width="112px" alt="location"></span>
+                        <span>202 Grant raod<br>
+                            California, CA<br>
+                            396191
+                        </span>
+                </li>
+                <li>
+                    <span>
+                        <img src="img1/email-13762.png" width="112px" alt="location"></span>
+                        <span>helloworld@gmail.com</span>
+                </li>
+                <li>
+                    <span>
+                        <img src="img1/icons8-phone-512.png" width="112px" alt="location"></span>
+                        <span>310-354-903 </span>
+                </li>
+            </ul>
+            <ul class="sci">
+                <li><a href="#"><img src="img1/1.png" width="40px"></a></li>
+                <li><a href="#"><img src="img1/2.png" width="40px"></a></li>
+                <li><a href="#"><img src="img1/3.png" width="40px"></a></li>
+                <li><a href="#"><img src="img1/5.png" width="40px"></a></li>
+            </ul>
+          </div>
+        </div>
+            <div class="contactform">
+                <h2>Send message</h2>
+                <div class="formbox">
+                    <div class="inputbox w50">
+                        <input type="text" name="" required>
+                        <span>First name</span>
+                    </div>
+                    <div class="inputbox w50">
+                        <input type="text" name="" required>
+                        <span>last name</span>
+                    </div>
+                    <div class="inputbox w50">
+                        <input type="text" name="" required>
+                        <span>Email address</span>
+                    </div>
+                    <div class="inputbox w50">
+                        <input type="text" name="" required>
+                        <span>Mobile number</span>
+                    </div>
+                    <div class="inputbox w100">
+                        <textarea name="" required></textarea>
+                        <span>Write your message here</span>
+                    </div>
+                    <div class="inputbox w100">
+                       <input type="submit" value="send">
+                    </div>
+                </div>
+            </div>
```

Comment:
```suggestion
            <div class="contactform">
            <div class="new">
                <h2>Send message</h2>
                <div class="formbox">
                    <div class="inputbox w50">
                        <input type="text" name="" required>
                        <span>First name</span>
                    </div>
                    <div class="inputbox w50">
                        <input type="text" name="" required>
                        <span>last name</span>
                    </div>
                    <div class="inputbox w50">
                        <input type="text" name="" required>
                        <span>Email address</span>
                    </div>
                    <div class="inputbox w50">
                        <input type="text" name="" required>
                        <span>Mobile number</span>
                    </div>
                    <div class="inputbox w100">
                        <textarea name="" required></textarea>
                        <span>Write your message here</span>
                    </div>
                    <div class="inputbox w100">
                       <input type="submit" value="send">
                    </div>
                </div>
            </div>
            </div>
```
![image](https://user-images.githubusercontent.com/91013960/198896512-4f33a8f2-3d17-47d1-8888-ec067324b87b.png)

i have added only extra div with class new you contactform touches right side (you can see your own live link) by adding these changes your form will look like same as image above (after adding these suggestion) .
there are two more changes required in this regards see further comments...

Reply:
thanks, great suggestion.